src/main/webapp/app/shared/subject/sitesettings.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from "@angular/core";
+import {Observable} from "rxjs";
+import {SiteSettings} from "./subject.model";
+import {HttpClient} from "@angular/common/http";
+
+@Injectable({providedIn: 'root'})
+export class SiteSettingsService {
+
+    private siteSettingsResourceUrl = 'api/sitesettings';
+
+
+    constructor(private http: HttpClient) {
+    }
+
+    getSiteSettings(): Observable<SiteSettings> {
+        return this.http.get<SiteSettings>(`${this.siteSettingsResourceUrl}`);
```

Comment:
The `siteSettingsResourceUrl` is a string, so you can just put it as arguments without template string (`)
```
return this.http.get<SiteSettings>(this.siteSettingsResourceUrl);
```